doc/SpecifyingApplications/CreatingSequenceDiagrams/CreatingSequenceDiagrams.md

```diff
@@ -0,0 +1,160 @@
+
+
+
+### Preparation  
+
+* If not yet existing, create an _Issue_ for creating the _Sequence Diagrams_.  
+* Open a local feature branch for creating the _Sequence Diagrams_.  
+
+
+### General  
+
+* Check out the diagrams at other application repositories, like [AutomatedLinkAcceptanceProxy](https://github.com/openBackhaul/AutomatedLinkAcceptanceProxy/tree/develop/spec/diagrams) or [NotificationProxy](https://github.com/openBackhaul/NotificationProxy/tree/develop/spec/diagrams) for examples
+* Diagrams are typically only generated for individual parts, with one diagram per forwarding in most cases
+
+### Identify Which Diagrams to Specify  
+
+* Typically, each individual operation server (i.e. those services listed under *operation-servers/own-oam/basic/individual* and *operation-servers/basic/individual* from your ServiceList) is a starting point for a diagram
+* An operation server can also be starting point for more than just one diagram
+* There might also be diagrams for *embed-yourself* and *bequeath-your-data-and-die* sequences
+
+### Specifying a Sequence Diagram
+
+#### Assigning a Proper Name
+* Create a new file under the diagrams folder, and name it according to one of the following schemas:
+  * `<number>_<short-content-description>.plantuml`: e.g. *100_ReadingLtpStructure.plantuml*
+  * `<is|im><number>_<short-content-description>.plantuml`: if there are own diagrams for *im* and *is* operation servers (individual managment and individual service) add the *is* or *im* prefix; e.g. *im000_ListenToControllers.plantuml* or *is020_NotifiyingOfDeviceAlarms.plantuml*
+* The chosen name should sufficiently indicate what the purpose of the diagram is
+* The starting number refers the the sequence number of the uuid of the operation server the diagram is created for
+  * sometimes a diagram may combine multiple forwardings with the same process flow, but for different operation servers; in those cases a *x* may be used to wildcard the related part of the sequence number, e.g. in MWDI there is diagram *00x_CyclicOperationBasedDeviceListSync.plantuml*, which maps four separate forwardings
+
+#### Sequence Diagram Content
+
+Check how the already existing diagrams of other applications look like to get an idea for how your application's diagrams need to be created!
+
+The sequence diagram consists of several building blocks.   
+The picture shows an example from the NotificationProxy:  
+![Diagram_building_blocks](./pictures/createSeqDiagram_01.png) 
+
+**1: Diagram name = *.pgn* file name**  
+* right after the opening `@startuml` the diagram name has to be provided
+* this is the same string as in the filename without the *.plantuml* file ending
+* when the diagram is exported to *.png*, the created picture file will be named with the string provided here
+
+**2: Diagram Title = ForwardingName**
+* the title is what will be shown as title in the rendered diagram
+* here, the name of the forwarding has to be provided

```

Comment:
Could you please change from
"here, the name of the forwarding has to be provided"
to
"here, the name of the forwarding (and later callback in the OAS) has to be provided"